.travis.yml

```diff
@@ -28,17 +30,22 @@ env:
 
 before_install:
 - wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz
-- /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-- brew update && brew cask install slack
+- if [ "$TRAVIS_OS_NAME" = "osx" ]; then /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ; fi
+- if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update && brew cask install slack ; fi
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo snap install slack --classic ; fi
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash - ; fi
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get install nodejs ; fi
 - touch custom-dark-theme.css
 
 install:
 - tar xzf master.tar.gz && cd kcov-master && mkdir build && cd build
 - cmake -DCMAKE_INSTALL_PREFIX=${HOME}/kcov .. && make && sudo make install
 - cd ../.. && rm -rf kcov-master && mkdir -p coverage
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo ln -s ${HOME}/kcov/bin/kcov /usr/bin/kcov ; fi
 
 script:
-- sudo kcov coverage slack-dark-mode.sh
+# - if [ "$TRAVIS_OS_NAME" = "osx" ]; then sudo kcov coverage slack-dark-mode.sh; fi
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo kcov coverage snap-slack-dark-mode.sh; fi
 
```

Comment:
I think Travis is unhappy after all the coverage for `slack-dark-mode.sh` disappeared.

Reply:
nah nah, that was a test to make sure it was doing what i expected.  I'll add that back, sure, but the problem is i think with the linux version of `kcov`...

Reply:
Its simply either reporting 0% or not working properly..

Reply:
idk, my code works, i got the coverage as far as i could, just the apps don't want to report the coverage at all.

The script itself works fine.